backend/src/services/api/userService.ts

```diff
@@ -0,0 +1,12 @@
+import * as userRepository from '../../data/repositories/userRepository';
+import { User } from '../../../../shared/src/interfaces/user.interface';
```

Comment:
Firstly you need to import it to the common folder in the `backend` and then use it from there
(don't forget about aliases)